include/pybind11/detail/internals.h

```diff
@@ -148,20 +148,35 @@ struct override_hash {
 
 using instance_map = std::unordered_multimap<const void *, instance *>;
 
+#ifdef Py_GIL_DISABLED
+// Wrapper around PyMutex to provide BasicLockable semantics
```

Comment:
Sorry I missed this when I looked before:

Could you please remove the

```
#include <mutex>
```

near the top? (I double-checked that `std::mutex` is not referenced anymore in this file.)

Reply:
We still need it for `std::unique_lock`

Reply:
Oh, ok, thanks.